package/MDAnalysis/core/topologyattrs.py

```diff
@@ -409,6 +411,9 @@ def getattr__(atomgroup, name):
             raise AttributeError("'{0}' object has no attribute '{1}'".format(
                     atomgroup.__class__.__name__, name))
 
+    @deprecate(message="Instant selector AtomGroup['<name>'] or AtomGroup.<name> "
+               "is deprecated and will be removed in 1.0. "
+               "Use AtomGroup.select_atoms('name <name>') instead.")
     def _get_named_atom(group, name):
```

Comment:
We now issue a warning for `atoms.ChainIDs` or `atoms.segids`. I don't think that was the indented effect. @richardjgowers could you have a look please.